apps/src/templates/projects/NewProjectButtons.jsx

```diff
@@ -12,21 +12,28 @@ const DEFAULT_PROJECT_TYPES = [
   'gamelab'
 ];
 
+const YOUNG_PROJECT_TYPES = [
+  'playlab',
+  'artist',
+  'minecraft_designer',
+  'flappy'
+];
+
 const PROJECT_INFO = {
   'playlab': {
     label: i18n.projectTypePlaylab(),
     thumbnail: "/shared/images/fill-70x70/courses/logo_playlab.png"
   },
   'playlab_k1': {
-    label: i18n.projectTypePlaylab(),
+    label: i18n.projectTypePlaylabPreReader(),
```

Comment:
I'd check with Ryan on these. For the project widget in stage extras, levelbuilders would choose between the two level types but they were both supposed to both say "Play Lab", but it's possible he's fine with this change.

Reply:
I've got confirmation with Ryan that these strings should have "Pre-reader" on them going forward.